### ISSUE TITLE
Gather user feedback to improve content for accessibility acceptance criteria

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-design-system#1850]
## What

Gather user feedback to help improve our [accessibility acceptance criteria document](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md).

## Why

We're soon going to publish our accessibility acceptance criteria document. (See [#1781](https://github.com/alphagov/govuk-design-system/issues/1781).)

This issue, which we'll link to from the doc, will be a place where users can leave feedback and discuss things with us and each other.

It should also help with keeping all the comments in one place, and saving users from having to create issues themselves.

## Who needs to know about this

Technical Writer, Developers, Content Designer

## Done when we

- [x] publish the accessibility acceptance criteria doc (linking to this issue)
- [x] tell users (for example, on accessibility channels and cross-gov Slack) about the doc and this issue
- [ ] use the feedback we get to iterate the content

**Comment by EoinShaughnessy** [comment]
Should the description for this issue have a similar format to issues in our [GitHub backlog](https://github.com/alphagov/govuk-design-system-backlog)?

For example, instead of the content under **What** / **Why** / **Who needs to know**... :

> Use this issue to discuss the [Design System documentation on accessibility acceptance criteria](link).

**Comment by EoinShaughnessy** [comment]
As per @hannalaakso's and @trang-erskine's suggestion, it would be good, in a future iteration, to include a pre-filled example for users to download. For example, the [cookie banner accessibility testing](https://docs.google.com/spreadsheets/d/1ExFstuvy_F8T4MfU2AIoMJRZdXSe9RZC0NEDaCGiLS4/edit#gid=0).

However, before we do that, we'd need to:

- establish there's a user need
- decide on wording / a legend to supplement the current colour-coding (as it is, this doc is inaccessible)
